terraform/testcases/configmap_provider_http/otconfig.tpl

```diff
@@ -0,0 +1,37 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+  metricstransform:
+    transforms:
+      - include: ".*${testing_id}"
+        match_type: regexp
+        action: update
+        operations:
+          - action: add_label
+            new_label: "testingId"
+            new_value: "${testing_id}"
+
+exporters:
+  logging:
+    loglevel: debug
```

Comment:
Do we NEED this? This will generate a lot of extra log lines in the GHA workflow logs.

Reply:
this is logged inside the environment where the collector is running. we don't push those logs to the GHA workflow, do we?

Reply:
Good point, thank you for catching that.

Reply:
Isn't there a template value used for this in other config templates?

Reply:
It's used in this same config but the `loglevel` values are not the same for the `logging` exporter. I think the logging exporter uses something like `detailed` and something else instead of `debug`.